### DISCUSSION TITLE
Network addressing

### DISCUSSION THREAD

**Discussion by nash-sprd** [utmapp/UTM#3329] [UTM Help (macOS)]
I installed UTM, and managed to run and AMD64 OS on it. It's working properly, but I cannot see the IPv4 details of the guest machine. 
My usecase is running Docker Swarm in the VM, which I require the IP address to achieve.

(MacOS, M1 chip)

Is there something I'm not seeing?

**Comment by cpach** [comment]
In Linux you typically use `ip addr` to display the IP address.
